data/css/_colors.scss

```diff
@@ -1,3 +1,5 @@
+$brand-color: #005a71 !default;
```

Comment:
If we need more colors then let's add them in a consistent way and not bolt them on top of what we have. I guess I need more context around this. I'm afraid this is a perfect example of what I was talking about this morning, about "top-down" design that bypasses the default theme...

Let's check with the designers whether the intention is to add a "brand color" to the default theme, and what components it should be used in, how it is different from the "accent color", etc. If it's only intended to be used for the course theme, then we should (1) push back, because I'm not happy about having a theme that only applies to one preset and a preset that can only use one theme, (2) if that fails then we should move it into the course SCSS file and prefix it with the course name, something like `course-brand-color`.

For now, I would say change the paginate button to use the accent color, then override it in the course theme to use the brand color; then we can merge the pagination changes.

Also, if you need 10 colors, where are the other two? :stuck_out_tongue_winking_eye: